app/controllers/api/v1/talent_controller.rb

```diff
@@ -30,7 +30,7 @@ def show
   end
 
   def update
-    if talent.id != current_user.talent.id
+    if !current_user.admin? && talent.id != current_user.talent&.id
```

Comment:
Should it be or here?
```suggestion
    if !current_user.admin? || talent.id != current_user.talent&.id
```

Reply:
I don't think so. even if a user is not an admin they should be able to change their own profile. this was added in the case an admin wants to change another user, which is now possible (to change the profile_type)

Reply:
Yup, got it!